openshift/templates/api/api-deploy.bcreg-base.param

```diff
@@ -23,8 +25,8 @@ AGENT_ADMIN_PORT=8024
 SMTP_SERVER_ADDRESS=
 SMTP_SERVER_PORT=
 ADMIN_EMAIL=
-EMAIL_SUBJECT=
-PUBLIC_SITE_URL=
+EMAIL_SUBJECT=Invite from Issuer-Kit
+PUBLIC_SITE_URL=https://issuer-kit.pathfinder.gov.bc.ca/
```

Comment:
Correct URL?  `pathfinder.gov.bc.ca` specified in a few places.

Reply:
This parameter is only used when generating an email invite, so it is irrelevant. I have, however, removed it from the configuration since it is not used in order to make it more straightforward.